common/js/src/nacl-module/nacl-module.js

```diff
@@ -67,7 +67,7 @@ GSC.NaclModule = function(naclModulePath, type, logModulePath = false) {
    * @type {!goog.log.Logger}
    * @const
    */
-  this.logger = GSC.Logging.getChildLogger(
+  this.logger_ = GSC.Logging.getChildLogger(
```

Comment:
nit: This should either be marked private, or not have a trailing `_`
https://google.github.io/styleguide/jsguide.html#naming-non-constant-field-names